segmentstore/storage/src/test/java/io/pravega/segmentstore/storage/StorageTestBase.java

```diff
@@ -406,8 +406,7 @@ private String getSegmentName(int id, String context) {
             for (int j = 0; j < APPENDS_PER_SEGMENT; j++) {
                 byte[] writeData = String.format(APPEND_FORMAT, segmentName, j).getBytes();
 
-                // Append some garbage at the end to make sure we only write as much as instructed, and not the whole InputStream.
-                val dataStream = new SequenceInputStream(new ByteArrayInputStream(writeData), new ByteArrayInputStream(extraData));
+                val dataStream = new ByteArrayInputStream(writeData);
```

Comment:
Why remove this? This is exactly what we were trying to test for...

Reply:
Now that we match mocks to match server behavior the test breaks.

Reply:
Given that these classes will get rewritten anyway with PDP-34 and that even right now, nobody accesses them directly except via RollingStorage, I think we should be fine with this change.